src/Backoffice/Users/App/Integrations/pVerify/Requests/EligibilitySummary.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Users\App\Integrations\pVerify\Requests;
+
+use Lightit\Backoffice\Users\Domain\DataTransferObjects\pVerifyEligibilitySummaryRequestDTO;
+use Saloon\Http\Request;
+
+class EligibilitySummary extends Request
+{
+    public function __construct(
+        protected pVerifyEligibilitySummaryRequestDTO $pVerifyEligibilitySummary,
+    ) {
+    }
+
+    public function resolveEndpoint(): string
+    {
+        return '/EligibilitySummary';
+    }
+
+    public function body(): array
```

Comment:
should we have a `createDTOFromResponse()` or we don't know how that would look?

Reply:
After discussing that technique with Sergio, the conclussion was that those createFrom methods should not be used unless we have different data mapings for a DTO creation, in which case we should use a factory class.